### ISSUE TITLE
re::DOTALL not matching \n (\x0d)

### ISSUE THREAD

**Issue by xuboying** [xuboying/PCRE2Plus#1]
VERSION: a128925 and older
workaround: `(?:.|[\x0a\x0d])`

**Comment by xuboying** [comment]
```
Perl Behaviour: (Linux)
uname -a
Linux ubuntu 4.2.0-27-generic #32~14.04.1-Ubuntu SMP Fri Jan 22 15:32:26 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
perl -E 'say 1 if "\x0d\x0a"=~/^.*$/s'
1
```

**Comment by xuboying** [comment]
Fix in  6028d91
